exchanges/poloniex/poloniex_wrapper.go

```diff
@@ -674,7 +674,6 @@ func (e *Exchange) SubmitOrder(ctx context.Context, s *order.Submit) (*order.Sub
 	if err != nil {
 		return nil, err
 	}
-	// TODO: These are just here to allow a minimal POC on just the smart request type
```

Comment:
🚧 Change request:
You've removed the TODO, but not actually actioned it.
I'll provide a full diff for you to make it very clear what I'm asking for.